cwd/app.js

```diff
@@ -337,8 +340,13 @@
   const renderView = view => {
     console.log(`Rendering view: ${view.name}`);
 
+    // Removes highlight from previous gauge if any
+    gauge = document.getElementById(`gauge-${prevGauge + 1}`);
+    if(gauge) gauge.classList.remove('currentGauge');
+
     // Highlights the first gauge of each view
```

Comment:
I know this is not a change from you, but could you change the comment so that it reflects that we are only activating a single highlight?  To me, this implies that each view's first gauge will then be highlighted, but it seems in reality only one gauge is changing status.